cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2733,7 +2733,37 @@ action.cyences_notable_event_action.attacker_drilldown = `cs_o365` sourcetype="o
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
 
-
+[O365 - OneDrive and Sharepoint file sharing with external user]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 0 1 * * *
+description = This alert will pinpoint when file sharing with external user usein Onedrive and Sharepoint. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -2d@h
+dispatch.latest_time = +2d@h
```

Comment:
Check what is the time delay of actual file sharing happened and when you received event in splunk. based on that update earliest and latest time.

Also, include _index_earlies & _index_latest similar to other o365 alerts if you found delay in event ingestion.